README.md

```diff
@@ -1,4 +1,4 @@
-# Hutte Recipe - Remove Permission Assignments from Profiles
```

Comment:
@hmayerhutte Is there a reason to remove `Hutte Recipe - ` from the title?

Reply:
by accident - re-introduced now ✅